data/core/units/elves/Avenger.cfg

```diff
@@ -17,9 +17,9 @@
     {AMLA_DEFAULT}
     cost=53
     usage=mixed fighter
-    description= _ "The curious name of the elvish ‘Avengers’ comes from a tactic often employed by these master rangers. The enemy is allowed to break through a feint defense, and when the vulnerable troops behind the front line follow, these archers break cover and attack, cutting the supply lines and surrounding the enemy in one fell stroke.
+    description= _ "The curious name of the Elvish ‘Avengers’ comes from a tactic often employed by these master rangers. The enemy is allowed to break through a feint defense and when the vulnerable troops behind the front line follow, these archers break cover and attack, cutting supply lines and surrounding the enemy in one fell stroke. This has, at times, been interpreted as a form of vengeance for their brethren lost earlier in the battle. While not so base in design, it is not at all inaccurate.
```

Comment:
I wonder if maybe "defensive feint" is better than "feint defence"? Not sure though, if it changes the meaning that might be bad.

Reply:
I think it's okay as is.